src/neo/SmartContract/ApplicationEngine.Storage.cs

```diff
@@ -62,7 +62,11 @@ protected internal byte[] Get(StorageContext context, byte[] key)
 
         protected internal IIterator Find(StorageContext context, byte[] prefix, FindOptions options)
         {
-            if (options.HasFlag(FindOptions.KeysOnly) && options.HasFlag(FindOptions.ValuesOnly))
+            if ((options & (FindOptions)0xc0) != 0)
```

Comment:
I think we better define `FindOptions.All` and use it here, it's very easy to forget updating `0xc0` if/when we're to add another option.

Reply:
Done.